locales/pt_BR.json

```diff
@@ -0,0 +1,777 @@
+			"fail": "O 'landmark' deve ter um 'aria-label', 'aria-labelledby', ou 'title' único para tornar os 'landmarks' distinguíveis"
+		},
+		"has-lang": {
+			"pass": "O elemento <html> tem um atributo 'lang'",
+			"fail": "O elemento <html> não tem um atributo 'lang'"
+		},
+		"valid-lang": {
+			"pass": "O valor do atributo 'lang' está incluso na lista de idiomas válidos",
+			"fail": "O valor do atributo 'lang' não está incluso na lista de idiomas válidos"
+		},
+		"xml-lang-mismatch": {
+			"pass": "Os atributos 'lang' e 'xml:lang' têm o mesmo idioma base",
+			"fail": "Os atributos 'lang' e 'xml:lang' não têm o mesmo idioma base"
+		},
+		"dlitem": {
+			"pass": "O item da lista de descrição tem um elemento pai <dl>",
+			"fail": "O item da lista de descrição não tem um elemente pai <dl>"
+		},
+		"listitem": {
+			"pass": "O item de lista tem um elemento pai <ul>, <ol> ou role=\"list\"",
+			"fail": "O item de lista não tem um elemento pai <ul>, <ol> ou role=\"list\""
+		},
+		"only-dlitems": {
+			"pass": "O elemento de lista tem somente filhos diretos que são permitidos dentro de elementos <dt> ou <dd>",
+			"fail": "O elemento de lista tem filhos diretos que não são permitidos dentro de elementos <dt> ou <dd>"
+		},
+		"only-listitems": {
+			"pass": "O elemento de lista tem somente filhos diretos que são permitidos dentro de elementos <li>",
+			"fail": "O elemento de lista tem filhos diretos que não são permitidos dentro de elemento <li>"
+		},
+		"structured-dlitems": {
+			"pass": "Quando não vazio, o elemento tem ambos elementos <dt> e <dd>",
+			"fail": "Quando não vazio, o elemento não tem nem ao menos um elemento <dt> seguindo de pelo menos um elemento <dd>"
+		},
+		"caption": {
+			"pass": "O elemento multimídia tem uma trilha de legendas",
+			"incomplete": "Verifique se legendas estão disponíveis para o elemento"
+		},
+		"description": {
+			"pass": "O elemento multimídia tem uma trilha de audiodescrição",
+			"incomplete": "Verifique se a audiodescrição está disponível para o elemento"
+		},
+		"frame-tested": {
+			"pass": "O 'iframe' foi testado com o 'axe-core'",
+			"fail": "O 'iframe' não pôde ser testado com o 'axe-core'",
+			"incomplete": "O 'iframe' ainda tem que ser testado com o 'axe-core'"
+		},
+		"css-orientation-lock": {
+			"pass": "A exibição é operável, e o bloqueio da orientação não existe",
+			"fail": "Bloqueio de orientação CSS aplicado, o que torna a exibição inoperável",
+			"incomplete": "Bloqueio de orientação CSS não pode ser determinado"
+		},
+		"meta-viewport-large": {
+			"pass": "A tag <meta> não impede o zoom significativo em dispositivos móveis",
+			"fail": "A tag <meta> limita o zoom em dispositivos móveis"
+		},
+		"meta-viewport": {
+			"pass": "A tag <meta> não desabilita o zoom em dispositivos móveis",
+			"fail": "{{=it.data}} na tag <meta> desabilita o zoom em dispositivos móveis"
+		},
+		"header-present": {
+			"pass": "A página tem um cabeçalho (header)",
```

Comment:
Why include (header)?

Reply:
This is just like I comment before. The word 'cabeçalho' can be used both for header or heading, so it can be confusing. When saying just 'cabeçalho', someone could think that what is missing is a heading.